internal/alb/rs/rules.go

```diff
@@ -79,14 +82,12 @@ func NewDesiredRules(o *NewDesiredRulesOptions) (Rules, int, error) {
 	rs := o.ListenerRules
 	paths := o.Rule.HTTP.Paths
 
-	if len(paths) == 0 {
-		return nil, 0, fmt.Errorf("Ingress doesn't have any paths defined. This is not a very good ingress.")
+	if o.Priority == 0 {
+		o.Priority = 1
```

Comment:
seems hack to me 😄

Reply:
yeah i am actually not a fan of how the priority is built up. maybe it would be better to build all of the rules and then set the priorities after